### DISCUSSION TITLE
Add a play button underneath the thumbnail icon for each game

### DISCUSSION THREAD

**Discussion by awadallahfarris** [micah686/VnManager#33] [Ideas]
More of a minor feature request, but I think adding a little play button icon next to the manage icon underneath the thumbnail for each game would be greatly appreciated. It'd be nice to instantly boot the exe for the game right from the icon pages rather than having to click on the thumbnail first, then click on the play button.  Having this feature would reduce the need to make multiple clicks and make using the app to boot up your games much more snappy.

**Comment by superange128** [comment]
I agree that this would be a nice feature.

**Comment by micah686** [comment]
I'll see what I can do, but this might be one of the trickier issues. I'll need to keep track of the running game, and add some locking code to the main grid screen. (I currently have it so if you are playing a game, you can't navigate away from the game info/characters/screens tab)
